applications/liberland.md

```diff
@@ -0,0 +1,345 @@
+
+#### Incentive to HODL LLM   
+Due to the Proof of Commitment and the legislative system offering possible APR (see above and below respectively), token holders have an incentive to keep their LLM vested, and we expect this to raise and stabilize the price. 
+
+## Legislative Power 1: Representative Democracy   
+
+#### A Tokenized Parliament    
+States are internationally expected to run some form of representative democracy. The core of the legislative power implemented on our chain is Congress. We suggest other adopters will keep and adapt it.  
+The Congress is a customized pallet, using delegated voting functionality using the underlying Democracy pallet. Where the original Democracy pallet effects direct democracy, we focus on a faithful representation of a Representative democracy. By default, States and State-like actors currently tend towards this type of democracy, but it is underrepresented in current blockchain projects. 
+ 
+Representative democracy enacts laws and other regulations, both on-chain (as governance) and off-chain (for the state), and Congress is the heart of Liberland's political system. 
+
+#### Smart Elections   
+We implement an election system where a Citizen votes with their tokens by Liberland's design for a proof-of-stake-like meritocracy. The Citizens are analogous to Nominators. Instead of Validators, one chooses one or more Members of Congress to delegate their political power. 
+Members of Congress receive a share of LLM minted into the Treasury, and they are free to incentivize the voters to vote for them. Voters can expect (also) material returns for giving their trust to politicians, further strengthening the nPoS-like mechanics. 
+ 
+However, this system is easily adaptable to "one-citizen-one-vote" if required by more conservative adopters and without the voter incentive. Conversely, more experimental polities can tweak it and add alternative voting or other algorithms. The executive typically keeps the lion's share of the Treasury in those use-cases.
+ 
+In Liberland, Voters will be allowed to rethink their vote, even mid-term. Based on their performance, they can rearrange the LLM allotted to the Congressmen, creating live feedback to keep them loyal to their voter base.
+
+
+## Legislative Power 2: Direct democracy
+To fit the Liberland specs and enable more widespread adoption by traditional polities, we implement direct democracy. There, the system counts KYCed voters on the regular "one person-one vote" principle to introduce equality and equity between the rich and the poor (in Liberland, those who commit less and those who commit more).     
+
+Direct democracy is our basic check-and-balance: the people can second-guess their representatives in Congress. In Liberland, this is used to strike down unwanted regulations but can be easily adapted for positive democracy or even as the basis for elections. We complement this system with a front end.   
```

Comment:
Kind of :) 
Voters are ALWAYS free to practice direct NEGATIVE democracy themselves. Your frustration, anger, knowledge, erudition and economic situation is more than sufficient to correlate the right incentive/skin in the game to decide what Law/Regulation/Policy is NOT good for you.
They can also play the "spot the crook" game by practising Dismissal; that is, public veto for elected officials, such as the judges. We trust that people will be able to define who in their mind has the over-critical amount of bad press, suspicious deals and personal affairs that they might want them no longer to represent Liberland.

If you want to engage in POSITIVE direct democracy, you join with others, form a party. Then you get your person voted in. Unlike anywhere else in the world, you have that person on an "algorithmic (on-chain) leash" - if they break your trust, they lose your confidence. It's over for them now (losing power to enact proposals) and it will sure be over come the next elections (in three months!).
Hence, our representative democracy is more direct than practically any direct democratic system. It is also really representative. As I write in the Grant Application, in today's states, you can't give any binding feedback mid term to a member of parliament who broke your trust. You are, in fact, choosing your masters, not your representatives. In liberland, they will represent you for real, and will get "slashed" by you if not.

I would just call what we do "democracy". Without adjectives. And that what other states do "something else. " :) Makes sense?

Reply:
Much clearer now, I was confused by the terms that apparently mean something else in the Liberland ecosystem than they do in existing countries. Thanks for clearifying!